src/ops/ops.go

```diff
@@ -77,6 +79,7 @@ func NewOps(logger *logrus.Logger, proxySet bool) Ops {
 
 // ExecPrivilegeCommand execute a command in the host environment via nsenter
 func (o *ops) ExecPrivilegeCommand(liveLogger io.Writer, command string, args ...string) (string, error) {
+	o.log.Infof("Executing privilege command: %s %s", command, args)
```

Comment:
Please remove this.
It  will spam the log

Reply:
done